interface/usergroup/user_info.php

```diff
@@ -41,50 +55,66 @@ function(data)
 </head>
 <body class="body_top">
 
-<span class="title"><?php echo xlt('Password Change'); ?></span>
-<br><br>
-
 <?php
 
 $ip=$_SERVER['REMOTE_ADDR'];
 $res = sqlStatement("select fname,lname,username from users where id=?",array($_SESSION["authId"]));
 $row = sqlFetchArray($res);
       $iter=$row;
 ?>
-<div id="display_msg">
-</div>
-<br>
-<FORM NAME="user_form" METHOD="POST" ACTION="user_info.php"
- onsubmit="top.restoreSession()">
-<input type=hidden name=secure_pwd value="<?php echo $GLOBALS['secure_password']; ?>">
-<TABLE>
-<TR>
-<TD><span class=text><?php xl('Full Name','e'); ?>: </span></TD>
-<TD><span class=text><?php echo htmlspecialchars($iter["fname"] . " " . $iter["lname"], ENT_NOQUOTES); ?></span></td>
-</TR>
-
-<TR>
-<TD><span class=text><?php xl('Username','e'); ?>: </span></TD>
-<TD><span class=text><?php echo $iter["username"]; ?></span></td>
-</TR>
-
-<TR>
-<TD><span class=text><?php xl('Current Password','e'); ?>: </span></TD>
-<TD><input type=password name=curPass size=20 value="" autocomplete='off'></td>
-</TR>
+<div class="container">
+   <div class="row">
+      <div class="col-xs-12">
+         <div class="page-header">
+            <h3><?php echo xlt('Password Change'); ?></h3>
+         </div>
+      </div>
+   </div>
+   <div class="row">
+      <div class="col-xs-12">
+        <div class='text-primary' id="display_msg"></div>
+      </div> 
+   </div>
 
-<TR>
-<TD><span class=text><?php xl('New Password','e'); ?>: </span></TD>
-<TD><input type=password name=newPass size=20 value="" autocomplete='off'></td>
-</TR>
-<TR>
-<TD><span class=text><?php xl('Repeat New Password','e'); ?>: </span></TD>
-<TD><input type=password name=newPass2 size=20 value="" autocomplete='off'></td>
-</TR>
+  <div class="row">
+     <div class="col-xs-12">
+<form method='post' action='user_info.php' class='form-horizontal' onsubmit='return update_password()'>
+<input type=hidden name=secure_pwd value="<?php echo $GLOBALS['secure_password']; ?>">
+        <div class="form-group">
+           <label class='control-label col-sm-2'><?php echo xlt('Full Name') . ":"; ?></label>
+           <label><?php echo text($iter["fname"]) . " " . text($iter["lname"]) ?></label>
```

Comment:
This would be better as a  disabled input element with a disabled class. This way it will lineup better with the password fields but still not allow the user to change their name.